src/store/mutations.ts

```diff
@@ -235,7 +235,7 @@ class Mutations {
     Vue.set(state.pipelines, state.currentPipelineName, pipeline);
     if (pipeline.length) {
       const firstStep = pipeline[0];
-      if (firstStep.name === 'domain') {
+      if (firstStep.name === 'domain' && typeof firstStep.domain === 'string') {
```

Comment:
What does happen to `state.currentDomain` when a pipeline starts with another pipeline? Does that have an influence on something else?
(Searching in the project, I have trouble finding any use for this `state.currentDomain`. It seems directly computed from the current pipeline to me, so I wonder if we can just remove it.)

Reply:
I'll check!

Reply:
I found that it is useful to set default currentDomain if pipeline diseappear so i think that it's better to keep it.
If i remove the condition, there is broken tests on the Ci ( not in local ). Due to Reference : string or Pipeline as commentary below.